config/routes.rb

```diff
@@ -7,10 +7,13 @@
   localized do
     resources :events,       only: [:index, :show]
     resources :certificates, only: :show
+    get '/profile',          to: 'profile#show'
+    patch '/profile',        to: 'profile#update'
+    get '/profile/edit',     to: 'profile#edit'
```

Comment:
I think to use like this, but we are talking about the current_user, so adding an `:id`to url don't make sense to me. What do you think? Or we can use the nickname too...